src/components/Header/Header.tsx

```diff
@@ -0,0 +1,140 @@
+import EES from "../../images/EES.svg";
+import MenuIcon from "../../images/EESMenuIcon.svg";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+interface Props {
+  screenWidth?: "mobile" | "desktop";
+}
+
+export function Header({ screenWidth }: Props) {
+  const [windowWidth, setWindowWidth] = useState<number>(window.innerWidth);
+  const [showMenuIcon, setShowMenuIcon] = useState<boolean>(false);
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+  const isMobile = screenWidth === "mobile";
+
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setWindowWidth(width); // Track window width
+      setShowMenuIcon(width < 800 ? true : false);
+    };
+    handleResize();
+    window.addEventListener("resize", handleResize);
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  return (
+    <>
+      <div
+        className={`flex justify-between items-center w-full h-[74px] px-[128px] py-[80px] bg-wheat
```

Comment:
128px seems overkill for padding. 
I don't remember if you recall around the US where the figma shows 200px on each side. That was because the screen size the designer showed was too big. It is our job to bridge the gap. 
![image](https://github.com/user-attachments/assets/826b0b8c-c866-4725-a60c-f5ed669d7e66)

Look at the above image at 800px, majority of the screen seems to be taken by empty space.

Maybe use margin auto on both sides to get margins that change with screen width.

Reply:
Also I was looking into ways to use margin auto with tailwind. One of the issues that I came across was:
1. There doesn't seem to be a way to apply margin auto in tailwind. https://tailwindcss.com/docs/margin To bridge this gap. I couldn't help but wonder if there was a calc operation to fix this, I didn't look to deep into this, but for future reference this is what that might look like - calc(100% - 10px); Here is two pages on how to use, in and outside of tailwind A. outside of tailwind https://medium.com/@anton.martyniuk/complete-guide-to-building-responsive-ui-with-css-calc-a787832dc8b3 B. inside of tailwind https://www.geeksforgeeks.org/how-to-use-calc-in-tailwind-css/
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/7b19bc8e-4f0e-4ec2-8b2f-6e24d79fea76">